### ISSUE TITLE
Spectra to be added

### ISSUE THREAD

**Issue by Askaniy** [Askaniy/TrueColorTools#1]
- [ ] Irregular satellites 
https://iopscience.iop.org/article/10.3847/1538-3881/aab49b
http://vizier.u-strasbg.fr/viz-bin/VizieR?-source=J/AJ/155/184
- [ ] 1998 KY26
https://echo.jpl.nasa.gov/asteroids/ostro+1999_ky26_science.pdf
- [ ] Orcus & Vanth
https://arxiv.org/abs/0910.4784

You can write your suggestions below

**Comment by SevenSpheres** [comment]
Did [this source](https://arxiv.org/abs/1109.3116) have enough data to get a color for Saffar/Ups And Ab? I think no?

**Comment by Askaniy** [comment]
I have seen this source, but I can't add it in the usual way as there is no data available at 550 nm. Anyway, I added it to the list

**Comment by SevenSpheres** [comment]
Kap And b maybe? https://arxiv.org/abs/2010.02928 (again I don't really know if it has the right data)

**Comment by SevenSpheres** [comment]
https://arxiv.org/abs/2011.07169 ?

**Comment by Askaniy** [comment]
I can't find any spectral data there…

**Comment by SevenSpheres** [comment]
Yeah okay...I just saw "exoplanet colors" mentioned on Discord, saw that the paper had "spectrum" and "optical" and posted it here without looking more closely

**Comment by SevenSpheres** [comment]
https://docs.google.com/spreadsheets/d/1kg9lktTwNhHVVyAS2FhccvLMnddI4L0Ot67f-kYFIFM/edit?usp=sharing

**Comment by Askaniy** [comment]
A suggestion to add 33434 Scottmanley and 4942 Munroe

**Comment by SevenSpheres** [comment]
Please add Ceres, Vesta, and maybe other large main-belt asteroids. SMASS should have data for them.

**Comment by Askaniy** [comment]
Done, I added SMASS spectra by pedro_jg.

**Comment by spaceguyidk** [comment]
Can you add 1998 KY26 from this paper? https://www.aanda.org/articles/aa/full_html/2026/05/aa59046-26/aa59046-26.html

**Comment by Askaniy** [comment]
@spaceguyidk added in https://github.com/Askaniy/TrueColorTools/commit/1b14785f11d36403b5440d4f9979291fffefc152, thanks for the suggestion!

<img width="1483" height="1018" alt="Image" src="https://github.com/user-attachments/assets/98c8d228-6641-43fe-b6f9-ba5e089ee2ae" />

**Comment by spaceguyidk** [comment]
No problem! I have actually made a list that has papers that could have spectra. And I mean with "could" is that i didn't really read the entire paper, Only a bit of the abstract. So sorry if there is some that can not be added due to not being spectra at all, Or just in Infrared or ultraviolet or beyond, Or may be already be in TrueColorTools.

[list of possible spectra.txt](https://github.com/user-attachments/files/30718906/list.of.possible.spectra.txt)

**Comment by spaceguyidk** [comment]
I have also found spectra of 2025 FA22, as you can see here: https://arxiv.org/pdf/2601.19770

**Comment by Askaniy** [comment]
Thank you for the list! I don't have much time right now, but I'm going to review it later
